int/config/network.go

```diff
@@ -155,6 +136,27 @@ func (n *NetworkManager) Network() *NetworkInfos {
 	return &n.networkInfos
 }
 
+func (n *NetworkManager) version(nodeURL string) (*string, error) {
+	client := node.NewClient(nodeURL)
+
+	status, err := node.Status(client)
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve node status: %w", err)
+	}
+
+	pattern := `.+\.(\d+\.\d+)`
+
+	re := regexp.MustCompile(pattern)
+	matches := re.FindStringSubmatch(*status.Version)
+
+	//nolint:gomnd
+	if len(matches) != 2 {
+		return nil, fmt.Errorf("failed to parse node version from: %s", *status.Version)
+	}
+
+	return &matches[1], nil
```

Comment:
Why not returning matches[1] instead ? this will make code easier to use/read since we won't have to dereference the string (version instead of *version)

Reply:
the only reason i do this it do be able to return `nil , fmt.Errorf("....` but yes maybe we dont care and i can return `"", fmt.Error` instead, i dont know what is the good practice here

Reply:
The good practice is to return the default value of the type, here "".